### PR TITLE
Note version is required for beta IGM version

### PR DESCRIPTION
**Pull request by emilymye**
Fixes https://github.com/terraform-providers/terraform-provider-google/issues/3045

-----------------------------------------------------------------
# [all]
## [terraform]
Note version is required for beta IGM version
### [terraform-beta]
Note version is required for beta IGM version

### CODE REVIEW

File: third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 1]
```diff
@@ -93,7 +93,7 @@ The following arguments are supported:
     which all new instances will be created. This field is only present in the
     `google` provider.
 
-* `version` - (Optional, [Beta](https://terraform.io/docs/providers/google/provider_versions.html)) Application versions managed by this instance group. Each
+* `version` - (Optional for GA, Required for [Beta](https://terraform.io/docs/providers/google/provider_versions.html)) Application versions managed by this instance group. Each
```
Comment (rileykarson) [inline third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown]: Our changes in `2.0.0` made `version` `Required` in `google-beta`, but actually removed it from `google` altogether; there, `instance_template` is `Required` while it's not present in `google-beta` at all (and has never been I don't think).

Comment (emilymye) [reply third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown]: 👍

File: third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 2]
```diff
@@ -93,7 +93,7 @@ The following arguments are supported:
     which all new instances will be created. This field is only present in the
     `google` provider.
 
-* `version` - (Optional, [Beta](https://terraform.io/docs/providers/google/provider_versions.html)) Application versions managed by this instance group. Each
+* `version` - (Required, [Beta](https://terraform.io/docs/providers/google/provider_versions.html)) Application versions managed by this instance group. Each
     version deals with a specific instance template, allowing canary release scenarios.
     Structure is documented below.
```
Comment (rileykarson) [inline third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown]:
```suggestion
    This field is only present in the `google-beta` provider. Structure is documented below.
```

**Comment by emilymye** [comment]
@rileykarson is instance template required though? in the GA provider it's marked as optional. The version.instance_template is actually marked as required in the docs (down below in the version block)

**Comment by rileykarson** [comment]
I believe it is! I think it being `Optional` is a holdover from when we supported `version` in the `google` provider. It's probably worth marking `Required` in the provider here too.

**Pull request merged**